php/content.md

```diff
@@ -56,15 +56,7 @@ $ docker build -t my-php-app .
 $ docker run -d --name my-running-app my-php-app
 ```
 
-We recommend that you add a custom `php.ini` configuration. `COPY` it into `/usr/local/etc/php` by adding one more line to the Dockerfile above and running the same commands to build and run:
-
-```dockerfile
-FROM %%IMAGE%%:7.2-apache
-COPY config/php.ini /usr/local/etc/php/
-COPY src/ /var/www/html/
-```
-
-Where `src/` is the directory containing all your PHP code and `config/` contains your `php.ini` file.
+We recommend that you add a `php.ini` configuration file, see [Configuration](#configuration) for details.
```

Comment:
Anchor links don't seem to work on Docker Hub (https://hub.docker.com/_/php/).

Any idea how this can be handled?

Reply:
Usually we just reference it in a more general sense as in "see the 'Configuration' section for details".